drivers/watchdog/wdt_sam0.c

```diff
@@ -143,23 +141,23 @@ static int wdt_sam0_install_timeout(struct device *dev,
 				/* Ensure we have time for the early warning */
 				per += 1;
 			}
-			WDT_REGS->EWCTRL.bit.EWOFFSET = per - 1;
+			WDT->EWCTRL.bit.EWOFFSET = per - 1;
```

Comment:
This should be `WDT->EWCTRL.bit.EWOFFSET = per - 1U;` now after 24d71431e9c9da493ce00eb284ac9b0b2cb1bdca

Reply:
ACK and rebased.